### PR TITLE
Fix rpm build by adding config to env variable

### PR DESCRIPTION
**Pull request by sruti1312**
Signed-off-by: sruti1312 <srutiparthiban@gmail.com>

**Is your feature request related to a problem? Please provide an existing Issue # , or describe.**
https://github.com/opensearch-project/performance-analyzer/issues/197

**Describe the solution you are proposing**
Adds config location to env variable

**Describe alternatives you've considered**

**Additional context**

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/performance-analyzer/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by dzane17 — APPROVED** [review]

File: packaging/rpm/postinst [hunk 1]
```diff
@@ -25,13 +30,13 @@ chown opensearch /var/lib/opensearch/rca_enabled.conf
...
-   echo >> /etc/opensearch/jvm.options
-   echo '## OpenSearch Performance Analyzer' >> /etc/opensearch/jvm.options
-   echo "-Dclk.tck=$CLK_TCK" >> /etc/opensearch/jvm.options
-   echo "-Djdk.attach.allowAttachSelf=true" >> /etc/opensearch/jvm.options
-   echo "-Djava.security.policy=file:///usr/share/opensearch/config/opensearch-performance-analyzer/opensearch_security.policy" >> /etc/opensearch/jvm.options
+   echo >> $OPENSEARCH_PATH_CONF/jvm.options
+   echo '## OpenSearch Performance Analyzer' >> $OPENSEARCH_PATH_CONF/jvm.options
+   echo "-Dclk.tck=$CLK_TCK" >> $OPENSEARCH_PATH_CONF/jvm.options
+   echo "-Djdk.attach.allowAttachSelf=true" >> $OPENSEARCH_PATH_CONF/jvm.options
+   echo "-Djava.security.policy=file:///etc/opensearch/opensearch-performance-analyzer/opensearch_security.policy" >> $OPENSEARCH_PATH_CONF/jvm.options
```
Comment (sgup432) [inline packaging/rpm/postinst:39]: /etc/opensearch/opensearc-performance/.... -> $OPENSEARCH_PATH_CONF/opensearch-performance-analyzer... ?

File: packaging/deb/postinst [hunk 2]
```diff
@@ -19,13 +24,13 @@ chown opensearch /var/lib/opensearch/rca_enabled.conf
...
-   echo >> /etc/opensearch/jvm.options
-   echo '## OpenSearch Performance Analyzer' >> /etc/opensearch/jvm.options
-   echo "-Dclk.tck=$CLK_TCK" >> /etc/opensearch/jvm.options
-   echo "-Djdk.attach.allowAttachSelf=true" >> /etc/opensearch/jvm.options
-   echo "-Djava.security.policy=file:///usr/share/opensearch/config/opensearch-performance-analyzer/opensearch_security.policy" >> /etc/opensearch/jvm.options
+   echo >> $OPENSEARCH_PATH_CONF/jvm.options
+   echo '## OpenSearch Performance Analyzer' >> $OPENSEARCH_PATH_CONF/jvm.options
+   echo "-Dclk.tck=$CLK_TCK" >> $OPENSEARCH_PATH_CONF/jvm.options
+   echo "-Djdk.attach.allowAttachSelf=true" >> $OPENSEARCH_PATH_CONF/jvm.options
+   echo "-Djava.security.policy=file:///etc/opensearch/opensearch-performance-analyzer/opensearch_security.policy" >> $OPENSEARCH_PATH_CONF/jvm.options
```
Comment (peterzhuamazon) [inline packaging/deb/postinst:33]: Why leave one /etc/opensearch out with hardcoding?

File: packaging/rpm/postinst (hunk 1 above)
Comment (peterzhuamazon) [reply packaging/rpm/postinst:39]: Same as above

Comment (sruti1312) [reply packaging/rpm/postinst]: Modified.

File: packaging/deb/postinst (hunk 2 above)
Comment (sruti1312) [reply packaging/deb/postinst]: Modified.

**Review by sgup432 — APPROVED** [review]

**Review by dzane17 — APPROVED** [review]

**Pull request merged**